ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/mgmt_interface_description.yml

```
@@ -8,4 +8,5 @@ l2leaf:
   nodes:
     - name: mgmt_interface_description
       mgmt_ip: 1.1.1.2
+      mgmt_gateway: 1.1.1.2
```

Use a different IP than the interface one (I know this is not valid config but from a networking standpoint it looks very wrong :) )

```suggestion
      mgmt_gateway: 1.1.1.254
```

---

Fixed